tests/cephfs/clients/mds_conf_modifying.py

```diff
@@ -56,6 +57,15 @@ def run(ceph_cluster, **kw):
                 )
                 return 1
 
+        if LooseVersion(ceph_version) >= LooseVersion("18.2"):
+            result = set_balance_automate_and_validate(client1)
+            if result:
```

Comment:
I think we don't need this if else block as we are directly raising the exception.
This will take program control to exception block where we are returning 1

Reply:
Ok, Removed the check